### DISCUSSION TITLE
Monday Due Date

### DISCUSSION THREAD

**Discussion by CarolineChiefari** [CarolineChief/Tafe240261#10] [General]
Hey,
Just wanted to check in how you guys are going? 
According to Stephen we should have submitted our feature branches and merged to the release branch by today so we can complete Part D, but I'm sure you guys are just as busy as I am working on all the assignments due next week. 
If you can, please check in so we as a group know how things a going and post any questions so we can hopefully help each other to get it all complete and ready to submit Monday. 
If anyone needs more time, we can always ask Stephen for an extension on the assignment, but please let the group know too so we are aware. 
Thanks
Caroline

**Comment by CarlaCunningham** [comment]
Hey, I'm going to ask for an extension because I'm pretty far behind.

**Comment by Hayden-Badenhop** [comment]
Yea im gonna talk to him monday as for my part all I have gotten is a screen shot I upload, but no formulas or anything to make it work. I sent him an email early this week asking but he hasn't seen it.

**Comment by CarolineChiefari** [comment]
Thanks for the update everyone, if you guys want, we can talk to him as a group in Monday's class.
